example/app/example-app/cronjobs.go

```diff
@@ -58,7 +60,13 @@ func InitCronJob(name string, params InitCronJobParams) projection_entity.CronJo
 		databaseURL := migrationhelper.GenerateDefaultDatabaseURL(name, connString)
 		migrationHelper := github_migrationhelper.NewGithubMigrationHelper(sourceURL, databaseURL)
 
-		return bridge_activity_matcher.New(params.Logger, params.RdbConn, migrationHelper)
+		config := bridge_activity_matcher.Config{}
+		err := config.Fill(params.ExtraConfigs[name])
```

Comment:
If the use of `Fill` is expected to be followed by first allocating an empt struct, how about make it a new method?

`bridge_activity_matcher.DecodeConfigFromInterface(params.ExtraConfigs[name])`

Reply:
Thanks for the advice, Updated